pkg/controller/pxc/upgrade.go

```diff
@@ -350,20 +352,29 @@ func (r *ReconcilePerconaXtraDBCluster) waitHostgroups(ctx context.Context, cr *
 	defer database.Close()
 
 	podNamePrefix := fmt.Sprintf("%s.%s.%s", pod.Name, sfsName, cr.Namespace)
```

Comment:
On the same file i.e. upgrade.go, we constract this prefix 4 times already, I believe we can make the implementation less error prone by just extracting this construction to a func.